tests/functions/functions.test.js

```diff
@@ -83,16 +87,67 @@ module.exports.handle = (event, context, cb) => {
     execSync(`${serverlessExec} deploy`);
   });
 
+  it('should create and deploy second function', async () => {
+    await sleep(30000);
```

Comment:
Do we need this wait here? Can we just run the second deploy straight away?

Reply:
removed